### ISSUE TITLE
`radiogroup-roving-tabindex` warnings during build

### ISSUE THREAD

**Issue by howard-e** [w3c/aria-at#905]
Noticed a warning when `radiogroup-roving-tabindex` is being evaluated, while running the `npm run build` script. The full console warning is detailed below in a collapsible section and the unique warnings are listed afterwards.

<details>
<summary>Expand warnings details:</summary>

```
*** 48 Errors in tests and/or commands in test plan [tests/radiogroup-roving-tabindex] ***
[Test 23]: Reference does not exist: aria-activedescendant
[Test 24]: Reference does not exist: aria-activedescendant
[Test 25]: Reference does not exist: aria-activedescendant
[Test 26]: Reference does not exist: aria-activedescendant
[Test 27]: Reference does not exist: aria-activedescendant
[Test 28]: Reference does not exist: aria-activedescendant
[Test 29]: Reference does not exist: aria-activedescendant
[Test 30]: Reference does not exist: aria-activedescendant
[Test 31]: Reference does not exist: aria-activedescendant
[Test 32]: Reference does not exist: aria-activedescendant
[Test 33]: Reference does not exist: aria-activedescendant
[Test 33]: Setup script does not exist: "setFocusOnFirstRadioButtonAndCheckSecondRadioButton" for task "navigate to the next checked radio button"
[Test 34]: Reference does not exist: aria-activedescendant
[Test 34]: Setup script does not exist: "setFocusOnFirstRadioButtonAndCheckSecondRadioButton" for task "navigate to the next checked radio button"
[Test 35]: Reference does not exist: aria-activedescendant
[Test 35]: Setup script does not exist: "setFocusOnSecondRadioButtonAndCheckFirstRadioButton" for task "navigate to the previous checked radio button"
[Test 36]: Reference does not exist: aria-activedescendant
[Test 36]: Setup script does not exist: "setFocusOnSecondRadioButtonAndCheckFirstRadioButton" for task "navigate to the previous checked radio button"
[Test 37]: Reference does not exist: aria-activedescendant
[Test 38]: Reference does not exist: aria-activedescendant
[Test 39]: Reference does not exist: aria-activedescendant
[Test 40]: Reference does not exist: aria-activedescendant
[Test 41]: Reference does not exist: aria-activedescendant
[Test 42]: Reference does not exist: aria-activedescendant
[Test 43]: Reference does not exist: aria-activedescendant
[Test 44]: Reference does not exist: aria-activedescendant
[Test 23]: Reference does not exist: aria-activedescendant
[Test 24]: Reference does not exist: aria-activedescendant
[Test 25]: Reference does not exist: aria-activedescendant
[Test 26]: Reference does not exist: aria-activedescendant
[Test 27]: Reference does not exist: aria-activedescendant
[Test 28]: Reference does not exist: aria-activedescendant
[Test 29]: Reference does not exist: aria-activedescendant
[Test 30]: Reference does not exist: aria-activedescendant
[Test 31]: Reference does not exist: aria-activedescendant
[Test 32]: Reference does not exist: aria-activedescendant
[Test 33]: Reference does not exist: aria-activedescendant
[Test 34]: Reference does not exist: aria-activedescendant
[Test 35]: Reference does not exist: aria-activedescendant
[Test 36]: Reference does not exist: aria-activedescendant
[Test 37]: Reference does not exist: aria-activedescendant
[Test 38]: Reference does not exist: aria-activedescendant
[Test 39]: Reference does not exist: aria-activedescendant
[Test 40]: Reference does not exist: aria-activedescendant
[Test 41]: Reference does not exist: aria-activedescendant
[Test 42]: Reference does not exist: aria-activedescendant
[Test 43]: Reference does not exist: aria-activedescendant
[Test 44]: Reference does not exist: aria-activedescendant

(31) out of (32) test plan(s) successfully processed and generated without any validation errors.
```

</details>

The unique warnings are:

- Tests 23-44 (`Reference does not exist: aria-activedescendant`)
- Tests 33 & 34 (`Setup script does not exist: "setFocusOnFirstRadioButtonAndCheckSecondRadioButton" for task "navigate to the next checked radio button"`)
- Tests 35 & 36 (`Setup script does not exist: "setFocusOnSecondRadioButtonAndCheckFirstRadioButton" for task "navigate to the previous checked radio button"`)

Seems the warnings first appeared in af9bb771a871b393fa9c9ad803ea52a1572e5d00.

**Comment by IsaDC** [comment]
Thank you, I'm working on fixing those issues.

**Comment by howard-e** [comment]
Resolved by #906
